src/views/News/index.js

```diff
@@ -134,415 +204,640 @@ export default class StudentNews extends Component {
+              <Grid
+                container
+                alignItems="baseline"
+                justify="center"
+                style={styles.searchBar}
+                spacing={5}
+              >
+                <Grid item xs={10} sm={8} md={8} lg={6}>
+                  <TextField
+                    id="search"
+                    label="Search news"
+                    value={search}
+                    onChange={(event) => {
+                      setSearch(event.target.value);
+                    }}
+                    margin="normal"
+                    fullWidth
+                  />
+                </Grid>
+              </Grid>
+            </Grid>
 
-            <Grid container justify="center">
-              {/* Search */}
-              <Grid item xs={12} md={12} lg={8}>
-                <Grid
-                  container
-                  alignItems="baseline"
-                  justify="center"
-                  style={styles.searchBar}
-                  spacing={5}
-                >
-                  <Grid item xs={10} sm={8} md={8} lg={6}>
+            {/* NOTE: leaving helper text for now in case
+            that is better than disabling submit button */}
+            {/* Create Posting */}
+            <Dialog open={openPostActivity} fullWidth>
+              <DialogTitle> Post on Student News </DialogTitle>
+              <DialogContent>
+                <Grid container>
+                  {/* CATEGORY ENTRY */}
+                  <Grid item>
+                    <TextField
+                      select
+                      label="Category"
+                      name="newPostCategory"
+                      value={newPostCategory}
+                      onChange={(event) => {
+                        setNewPostCategory(event.target.value);
+                      }}
+                      // helperText="Please choose a category."
+                      style={{ minWidth: '7rem' }}
+                    >
+                      {categories.map((category) => (
+                        <MenuItem key={category.categoryID} value={category.categoryID}>
+                          {category.categoryName}
+                        </MenuItem>
+                      ))}
+                    </TextField>
+                  </Grid>
+
+                  {/* SUBJECT ENTRY */}
+                  <Grid item xs={12}>
+                    <TextField
+                      label="Subject"
+                      margin="dense"
+                      fullWidth
+                      name="newPostSubject"
+                      value={newPostSubject}
+                      onChange={(event) => {
+                        setNewPostSubject(event.target.value);
+                      }}
+                      // helperText="Please enter a subject."
+                    />
+                  </Grid>
+
+                  {/* BODY ENTRY */}
+                  <Grid item xs={12}>
                     <TextField
```

Comment:
While you're working in this file, we oughta update the TextField's to use the `filled` variant:
```suggestion
                    <TextField
                      variant="filled"
```